src/app/page.tsx

```diff
@@ -6,6 +7,7 @@ export default function Home() {
     <>
       <Hero />
       <ProductList />
+      <LoginModal/>
     </>
   )
 }
```

Comment:
Login modal should be connected with cart button in the header with open close functionality